rpc/eth_api.go

```diff
@@ -247,6 +247,20 @@ func (e *PublicEthAPI) GetCode(address common.Address, blockNumber BlockNumber)
 	return out.Code, nil
 }
 
+// GetTxLogs returns the logs given a transaction hash.
+func (e *PublicEthAPI) GetTxLogs(txHash common.Hash) ([]*ethtypes.Log, error) {
+	// do we need to use the block height somewhere?
```

Comment:
sometimes is useful for clients to get the block height at the same time they perform the query. This change was introduced on the SDK because wallets had to query/subscribe to the txs and then query/subscribe to the blocks too.

Reply:
okay I see, would it be necessary to add it here?